### PR TITLE
Update Pollux Adapter to v1.0

### PR DESCRIPTION
**Pull request by hdjvieira**
## Type of change
- [x] Refactoring (no functional changes, no api changes)

## Description of change
Update Pollux Adapter to v1.0

- contact email of the adapter’s maintainer tech[at]polluxnetwork[dot]com
- [x] official adapter submission

## Other information
Related PR #1431

### CODE REVIEW

**Comment by hdjvieira** [comment]
Yeah @snapwich, sorry about that, created the pull request way too soon :)
Just pushed new adapter version

**Comment by mkendall07** [comment]
@hdjvieira Can you resolve the conflict

**Comment by mkendall07** [comment]
@snapwich 
Any update on review?

File: modules/polluxBidAdapter.js [hunk 1]
```diff
@@ -1,97 +1,115 @@
...
+  },
+  /**
+     * Make a server request from the list of BidRequests.
+     *
+     * @param {validBidRequests[]} - an array of bids
+     * @return ServerRequest Info describing the request to the server.
+     */
+  buildRequests: function (validBidRequests) {
+    if (!Array.isArray(validBidRequests) || !validBidRequests.length) {
+      return {};
```
[the remark this answers is not in the document]
Comment (hdjvieira) [reply modules/polluxBidAdapter.js]: @snapwich File changed, please review
Thank you

File: modules/polluxBidAdapter.js [hunk 2]
```diff
@@ -1,97 +1,115 @@
...
-      bidObject.cpm = parseFloat(response.cpm);
-      if (response.ad_type === 'url') {
-        bidObject.adUrl = response.ad;
+    // loop through serverResponses
+    const bidResponses = [];
+    for (var b in serverResponse) {
+      var bid = serverResponse[b];
+      const bidResponse = {
+        requestId: bid.bidId, // not request id, it's bid's id
+        bidderCode: spec.code,
```
Comment (mkendall07) [inline modules/polluxBidAdapter.js]: Please drop `bidderCode` here. It's not required and in fact breaks adapter aliasing. Thanks sorry for the confusion.

Comment (hdjvieira) [reply modules/polluxBidAdapter.js]: Will do, but just enlight me here :)
"bidderCode" was a required parameter before, or at least it was in the [docs](http://prebid.org/dev-docs/bidder-adapter-1.html#interpreting-the-response) right?
Thanks for your help

Comment (mkendall07) [reply modules/polluxBidAdapter.js]: Right on both counts. We have since fixed the documentation to omit that param. Thanks

**Pull request merged**